storm-server/src/main/java/org/apache/storm/scheduler/resource/ResourceAwareScheduler.java

```diff
@@ -42,14 +43,16 @@
     private Map<String, Object> conf;
     private ISchedulingPriorityStrategy schedulingPrioritystrategy;
     private IEvictionStrategy evictionStrategy;
+    private IConfigLoader configLoader;
 
     @Override
     public void prepare(Map<String, Object> conf) {
         this.conf = conf;
         schedulingPrioritystrategy = (ISchedulingPriorityStrategy) ReflectionUtils.newInstance(
-            (String) conf.get(DaemonConfig.RESOURCE_AWARE_SCHEDULER_PRIORITY_STRATEGY));
+                (String) conf.get(DaemonConfig.RESOURCE_AWARE_SCHEDULER_PRIORITY_STRATEGY));
```

Comment:
is there a reason why we need to change the spacing in many places in this file?

Reply:
No.. It just accidentally happened..